common/djangoapps/edeos/utils.py

```diff
@@ -55,6 +58,23 @@ def is_valid_edeos_field(fields):
             return False
     return True
 
+def get_user_id(user_model):
+    # type: (User) -> str
```

Comment:
This should probably be removed too
(here and in other places)

Reply:
Hm it is doc-type notation. It used to improve type inference, so it is can be left.  But as it used only in this place - maybe you right.